src/common/marketUtils.es6

```diff
@@ -11,6 +11,26 @@ const getMarketPosition = (() => {
     };
 })();
 
+const sortMarketSubmarkets = ([...markets]) => {
+    return markets.map((market) => {
+        if (market.submarkets) {
+            const submarkets_sort = market.submarkets.sort((a, b) => {
+                if (a.display_name < b.display_name) return -1;
+                if (a.display_name > b.display_name) return 1;
+            });
+            market.submarkets = submarkets_sort;
+        }
+        return market
+    })
+}
+
+const getMarketChildSorted = (markets) => {
```

Comment:
This function sorts markets and submarkets?

Reply:
yes, that is the compilation of sorting the object, which sorts market and submarkets. which we will export

Reply:
but the name should be change to `getMarketSubmarketsSorted` i think

Reply:
let me rename all the function to be more descriptive